src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -133,9 +133,10 @@ var inputValueTracking = {
       return false;
     }
     var tracker = getTracker(subject);
+    const isNode = (subject: any).nodeType === ELEMENT_NODE;
 
     if (!tracker) {
-      if (typeof (subject: any).tag === 'number') {
+      if (isNode || typeof (subject: any).tag === 'number') {
```

Comment:
Which is to say the `track` path is really more of a stack only path but not well named